code/modules/shuttle/navigation_computer.dm

```diff
@@ -119,7 +119,7 @@
 		return
 
 	if(!my_port)
-		my_port = new(locate(eyeobj.x - x_offset, eyeobj.y - y_offset, eyeobj.z))
+		my_port = new /obj/docking_port/stationary()
```

Comment:
`my_port = new`

Reply:
There is no reason to not be explicit

Reply:
No there is. Avoid changes that require future changes to be made in multiple locations

Reply:
I find `new` type inference to be ugly and uninformative. Using it is not in the contribution guidelines and I avoid using it as much as possible because it obfuscates what is actually being instanciated.

If someone wants to change the type of my_port, they can control+f for the single place it is assigned to.

Reply:
Making changes according to maintainer reviews however, is.

Feel free to find another maintainer who cares enough to say otherwise but being required to search up how things are used in various places in the code is one of the signs of unmaintainable code.

Reply:
The fact that using type inference is not required by the coding standard means that you will have to look up whether or not this variable uses it anyway.